services/object/src/api.js

```diff
@@ -10,4 +11,25 @@ let builder = new APIBuilder({
   context: ['cfg', 'db', 'backends'],
 });
 
+builder.declare({
+  method: 'post',
+  route: '/upload/:name/:projectId',
+  name: 'uploadObject',
+  input: undefined,
+  output: undefined,
+  stability: 'experimental',
+  category: 'Backend',
+  scopes: 'object:upload:<name>/<projectId>',
```

Comment:
This scope pattern would cause issues when we want to allow `*` patterns in the name, as the `*` would swallow up the projectId.